spanish-gitconfig

```diff
@@ -2,13 +2,21 @@
   agregar = add
   bitacora = log
   clonar = clone
+  control = remote
```

Comment:
You got me here. Please, explain.

Reply:
Debatable but, for example, if the TV remote is missing, someone might ask "have you seen the remote?" or "dónde quedó el control?" in Spanish.

It's a very tongue-in-cheek alias. A more straightforward one would of course be `remoto`, or `lejano`/`distante`.

Reply:
I'm down to add it to the soon to be available `AR-es` version, and use `distante` instead. I'll make the changes soon. 

This was great.

Reply:
Ok, i can change it to `distante` here. Is there anything else you'd like this PR to include? (I was lazy and did the first commit editing the file through the Github web interface)